build/configs/artik053/tc/defconfig

```diff
@@ -1091,6 +1091,7 @@ CONFIG_TC_KERNEL_ENVIRON=y
 CONFIG_TC_KERNEL_ERRNO=y
 # CONFIG_TC_KERNEL_GROUP is not set
 CONFIG_TC_KERNEL_LIBC_LIBGEN=y
+CONFIG_TC_KERNEL_LIBC_FIXEDMATH=y
```

Comment:
It is better to split Kconfig & defconfig changes to separate  patches.
1. One Patch for New Kconfig,
2. Updating defconfig to enable "FixedMath"